<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ApePlayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabExplorer">
       <attribute name="title">
        <string>Explorer</string>
       </attribute>
       <layout class="QVBoxLayout" name="explorerLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>6</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <attribute name="headerStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Item</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Info</string>
            </property>
           </column>
          </widget>
          <widget class="QWidget" name="rightPanel" native="true">
           <layout class="QVBoxLayout" name="vLayoutRight">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="waveformFrame">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #222; border: 1px solid #444; border-radius: 4px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <layout class="QVBoxLayout" name="waveformLayout"/>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="grpControls">
              <property name="title">
               <string>Preview Control</string>
              </property>
              <layout class="QHBoxLayout" name="ctlLayout">
               <item>
                <widget class="QPushButton" name="btnPlay">
                 <property name="text">
                  <string>Play</string>
                 </property>
                 <property name="icon">
                  <iconset theme="media-playback-start"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnStop">
                 <property name="text">
                  <string>Stop</string>
                 </property>
                 <property name="icon">
                  <iconset theme="media-playback-stop"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkLoop">
                 <property name="text">
                  <string>Loop Playback</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="grpInfo">
              <property name="title">
               <string>Structure Details</string>
              </property>
              <layout class="QVBoxLayout" name="infoLayout">
               <item>
                <widget class="QTableWidget" name="propTable">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Property</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Value</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabExport">
       <attribute name="title">
        <string>Export / Render</string>
       </attribute>
       <layout class="QVBoxLayout" name="vLayoutExport">
        <item>
         <widget class="QGroupBox" name="grpWav">
          <property name="title">
           <string>WAV Render</string>
          </property>
          <layout class="QGridLayout" name="gridWav">
           <item row="0" column="0">
            <widget class="QLabel" name="lbSq">
             <property name="text">
              <string>Sequence File (SQ/MID):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_sqPath">
             <property name="placeholderText">
              <string>Select a sequence file...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="btnBrowseSq">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="document-open"/>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="chkReverb">
             <property name="text">
              <string>Enable generic reverb processing</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnRenderWav">
             <property name="text">
              <string>Render WAV</string>
             </property>
             <property name="icon">
              <iconset theme="media-record"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpBulk">
          <property name="title">
           <string>Operations</string>
          </property>
          <layout class="QVBoxLayout" name="bulkLayout">
           <item>
            <widget class="QLabel" name="lbBulk">
             <property name="text">
              <string>Convert entire directory of .HD files to .SF2 SoundFonts and .SQ files to MIDI.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBulk">
             <property name="text">
              <string>Start Bulk Export...</string>
             </property>
             <property name="icon">
              <iconset theme="system-run"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSeq2Midi">
             <property name="text">
              <string>Convert SQ to MIDI...</string>
             </property>
             <property name="icon">
              <iconset theme="audio-x-generic"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpLog">
          <property name="title">
           <string>Output Log</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="logOutput">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string>Ready.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_HD"/>
    <addaction name="actionExportSF2"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_HD">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open .HD File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExportSF2">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Export Current to SF2...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About ApePlayer...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
